lib/ui/pages/fw/fw_intro.dart

```diff
@@ -38,8 +38,8 @@ class FwIntroPage extends StatelessWidget {
         OnboardingButton(
             label: "Continue",
             onTap: () {
-              if (UpdatesManager().getStoredFwVersion() == null) {
-                showBlurDialog(
+              if (UpdatesManager().getStoredFwVersion() != null) {
```

Comment:
This ˙!=` change left from testing?

Reply:
sorry about that. removed that change the only change in fw_intro is the showBlurDialog to showEnvoyDialog. fixed in  [56c8972](https://github.com/Foundation-Devices/envoy/pull/69/commits/56c89726e9d9f24701f1ee6fbd04d0f60aff0880)